primitives/core/src/offchain/mod.rs

```diff
@@ -36,6 +36,9 @@ pub trait OffchainStorage: Clone + Send + Sync {
 	/// Persist a value in storage under given key and prefix.
 	fn set(&mut self, prefix: &[u8], key: &[u8], value: &[u8]);
 
+	/// Remove a perviously pin storage ersisted value under given key and prefix.
```

Comment:
Typos. Not sure what "pin storage persisted value" is; if that was really what was meant, then a bit more documentation explaining the phrase would be helpful.

```suggestion
	/// Remove a previously storage-persisted value under given key and prefix.
```